package.json

```diff
@@ -17,14 +17,33 @@
     "debug": "~2.0.0",
     "ejs": "~0.8.5",
     "express": "~4.9.0",
-    "gulp": "^3.8.10",
-    "gulp-minify-css": "^0.4.5",
-    "gulp-replace": "^0.5.2",
-    "gulp-uglify": "^1.1.0",
     "mongoose": "^3.8.22",
     "morgan": "~1.3.0",
     "octonode": "^0.6.14",
     "serve-favicon": "~2.1.3",
     "validator": "^3.28.0"
+  },
+  "devDependencies": {
+    "browser-sync": "^2.0.1",
+    "browserify": "^8.1.3",
+    "exorcist": "^0.1.6",
```

Comment:
Also we should remove any dependecies that we aren't using so exorcist, gulp-browserify, gulp-nodemon, minifyify, uglifyify, browser-jsonp, vinyl-source-stream.